tests/functional/autoprompt/test_doc.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from awscli.clidriver import create_clidriver, ServiceCommand
+from awscli.customizations.autoprompt.doc import (
+    AwsTopLevelDocsGetter, DocsGetter, ServiceCommandDocsGetter,
+    ServiceOperationDocsGetter
+)
+from awscli.testutils import unittest
+
+
+class TestDocsGetter(unittest.TestCase):
+    def setUp(self):
+        self.driver = create_clidriver()
+        self.docs_getter = DocsGetter(self.driver)
+
+    def test_get_service_command_docs(self):
+        command_text = 'ec2'
+        actual_docs = self.docs_getter.get_docs(command_text)
+        expected_docs = (
+            '\n\nAmazon Elastic Compute Cloud (Amazon EC2) provides secure and'
+            '\nresizable computing capacity in the AWS cloud. Using '
```

Comment:
Ok so for all of these tests where we assert the exact sentences is really fragile because the documentation is liable to changing and breaking our tests suddenly. I'd recommend updating these to either:

1. Assert something very minimal but identifiable for that command like `Amazon EC2` and `Available Commands` are in the `expected_docs`
2. Create our own service model and keep our really specific assertions on that since we can guarantee that the service model will not change.